Products/Zuul/catalog/model_catalog_tool.py

```diff
@@ -270,6 +271,28 @@ def getValue(obj):
         return sorted((unbrain(brain) for brain in queryResults),
                       key=getValue, reverse=reverse, cmp=natural_compare)
 
+    def get_all_docs(self):
```

Comment:
There are some extra requirements needed here.  It needs to accept the `types`, `limit`, and `fields` parameters.  I plan to use this in the ZenETL zenpack, but I need this API to support these three parameters.

Also, with the addition of those parameters, `search_with_cursor` or `cursor_search` will probably be a better name for the method.